### DISCUSSION TITLE
ANTV [VE]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#394] [Show and tell]
### Channel ID

ANTV.ve

### Stream URL

http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x89ysyb

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Thank you for the stream: it will be included in the next release.

**Comment by freearhey** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/video/x89ysyb (offline)

**Comment by ghost** [comment]
the video is private
